blurring_as_a_service/settings/settings_schema.py

```diff
@@ -24,6 +24,7 @@ class MetadataPipelineSpec(SettingsSpecModel):
 class PerformanceEvaluationPipelineSpec(SettingsSpecModel):
     inputs: Dict[str, str] = None
     outputs: Dict[str, str] = None
+    experiment_name: str = None
```

Comment:
How come we want the experiment name to be in the config?

Reply:
Oh this is not the Azure experiment, maybe we can choose another name or improve it so it doesn't get confuse with the AzureML experiment?